### ISSUE TITLE
Installing version 2.3.0 = npm ERR! cb() never called!

### ISSUE THREAD

**Issue by keitharm** [vvo/gifify#92]
It looks like version 2.3.0 of gifify is broken when you try to install it via npm or yarn.

`$ npm i gifify -d`:
```bash
npm info it worked if it ends with ok
npm info using npm@3.10.10
npm info using node@v6.10.1
npm info attempt registry request try #1 at 1:13:35 PM
npm http request GET https://registry.npmjs.org/gifify
npm http 304 https://registry.npmjs.org/gifify
npm info addNameTag [ 'gifify', 'latest' ]
npm ERR! cb() never called!
```

`$ yarn add gifify`:
```
yarn add v0.21.3
info No lockfile found.
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/gifify/-/gifify-2.3.0.tgz: invalid tar file".
info If you think this is a bug, please open a bug report with the information provided in "/Users/keith/Desktop/t/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

I downloaded versions 2.2.2 and 2.3.0 and extracted the archives manually to see if I could find the problem and it looks like the only difference between the 2 versions (other than the src code) is a missing yarn.lock file and .npmignore.

`2.2.2`:
```bash
2.2.2
├── .npmignore   <---
├── HISTORY.md
├── LICENSE.md
├── README.md
├── bin
│   └── gifify
├── index.js
├── package.json
└── yarn.lock    <---

1 directory, 8 files
```

`2.3.0`:
```bash
2.3.0
├── HISTORY.md
├── LICENSE.md
├── README.md
├── bin
│   └── gifify
├── index.js
└── package.json

1 directory, 6 files
```

I found this reported issue on yarn's github repo: https://github.com/yarnpkg/yarn/issues/2764.

It looks like publishing via yarn breaks things for some reason currently and it can be fixed by deleting the old yarn file and regenerating it (https://github.com/yarnpkg/yarn/issues/2764#issuecomment-283669792)

**Comment by ztz0223** [comment]
Yes, I have this issue also, and I must set version as 2.2.2.
Could you be kind to fix this?
Thanks.

**Comment by vvo** [comment]
Just released 2.4.0, happy gifying, sorry for the mess.

**Comment by vvo** [comment]
![image](https://cloud.githubusercontent.com/assets/123822/24334092/6857270a-1264-11e7-91e0-f5bcd28471c0.png)
